vcxagency-node/src/service/notifications/service-new-messages.js

```diff
@@ -40,6 +40,35 @@ module.exports.createServiceNewMessages = function createServiceNewMessages (red
     redisClientRw.quit()
   }
 
+  redisClientRw.on('error', function (err) {
```

Comment:
I like new events added, but I don't like the move - these event subscriptions should be registered as soon as the redis clients are created. By moving it's file this requirement would be hard to keep in mind and maintain. I'd suggest rather some sort of "redis factory" which would created redis client and register these subscriptions as well.